MetaMask/metamask-extension
# Fix stringified object on NFT approve screen

## Explanation
Fixes a stringified object on the NFT approve screen. Since the title is a React object and not a string in case of "unknown NFTs" we need to render it as a React object instead of a string.

## Screenshots/Screencaps



### Before

![image](https://user-images.githubusercontent.com/1561200/179954003-32c5b2ef-7f91-4564-b9c2-068eaf2f7d0c.png)

### After

![image](https://user-images.githubusercontent.com/1561200/179953225-d32deb3b-608c-4aad-8450-7673bdf4e711.png)

## Manual Testing Steps
1. Go to https://etherscan.io/address/0x1e52F7A450b08b1B249A4f4f54518fC5278C2285#writeContract
2. Create a "setApprovalForAll" transaction to any address
3. See that no stringified objects are present on the screen

## Pre-Merge Checklist


### + If there are functional changes:
